wagtail/project_template/project_name/settings.py

```diff
@@ -125,7 +107,9 @@
     'compressor.finders.CompressorFinder',
 )
 
-MEDIA_ROOT = join(PROJECT_ROOT, 'media')
+STATIC_ROOT = os.path.join(BASE_DIR, 'static')
```

Comment:
Why isn’t `STATIC_ROOT` with `STATIC_URL` anymore? A more logic order would be:

1. STATICFILES_FINDERS
2. STATIC_ROOT & STATIC_URL
3. MEDIA_ROOT & MEDIA_URL